### PR TITLE
Initial data for questions

### PR DESCRIPTION
**Pull request by sophieq**

### CODE REVIEW

File: src/initialStore.js [hunk 1]
```diff
@@ -1,7 +1,461 @@
 const initialStore = {
```
Comment (JerryXie98) [inline src/initialStore.js:1]: Is currently not imported anywhere. Should probably build out a feature to store this information in a JSON file and parse it into the questionnaire.

File: src/components/Header.jsx [hunk 2]
```diff
@@ -10,7 +11,7 @@ class Header extends Component {
     return (
       <div className="header-container">
         <div>
-          <img className="logo" src="/logo.png" alt="" />
+          <img className="logo" src={`/${logo}`} alt="Logo" />
```
Comment (suzyng83209) [inline src/components/Header.jsx:14]: LOL you had to do this to get it to work? lmao that is very unfortunate.

**Review by suzyng83209 — APPROVED** [review]
some minor comments just confirm w me

File: webpack.config.js [hunk 3]
```diff
@@ -28,7 +28,14 @@ module.exports = {
       {
         test: /\.(png|svg|jpg|jpeg|gif|ico)$/,
         exclude: /node_modules/,
-        use: ["file-loader?name=[name].[ext]"] // ?name=[name].[ext] is only necessary to preserve the original file name
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: 'images/[name]-[hash:8].[ext]'
```
Comment (suzyng83209) [inline webpack.config.js:35]: did you not need "/'?

File: src/components/Header.jsx (hunk 2 above)
Comment (JerryXie98) [reply src/components/Header.jsx:14]: It be like sometime.

**Pull request merged**